packages/babel-core/src/transformation/store.js

```diff
@@ -1,11 +1,6 @@
 export default class Store {
```

Comment:
So this is literally a wrapper for Map now haha (also nice that `dynamicData` thing has already been a bit 😱 to me)

Reply:
Yeah we should probably just kill this file and copy this logic to the two classes where it's actually used, but I opted to leave that for a later PR.